File: easy/pro-duct-tape/DuctTape.py
import math

height = int(input())
width = int(input())
rollSquare = 10 # 60ft long * 2" / 12" width
doorSquare = 2 * height * width # Both sides of the door!

rollsRequired = math.ceil(doorSquare / rollSquare)
print(rollsRequired)

File: easy/pro-duct-tape/DuctTape.java
public final class DuctTape {
	public static void main(String[] args) { 
	    final var scanner = new java.util.Scanner(System.in);
	    final var doorHeight = scanner.nextInt();
	    final var doorWidth = scanner.nextInt();
			
	    final var rollSquare = 10; // 60ft long * 2" / 12" width
	    final var doorSquare = 2 * doorHeight * doorWidth; // Both sides of the door!
	    final var rollsRequired = (int) Math.ceil((double) doorSquare / rollSquare);
	    
	    System.out.print(rollsRequired);
	}
}
